.github/workflows/build_rpms.yaml

```diff
@@ -26,17 +26,17 @@ jobs:
       run: |
         echo "REGISTRY_USERNAME=${{ secrets.DOCKER_REGISTRY_USERNAME }}" >> $GITHUB_ENV
         echo "REGISTRY_PASSWORD=${{ secrets.DOCKER_REGISTRY_PASSWORD }}" >> $GITHUB_ENV
-    - name: Build and push RPMs
+    - name: Prepare options file
       run: |
-        echo $REGISTRY_PASSWORD | docker login docker.io --password-stdin --username $REGISTRY_USERNAME
-        mkdir /tmp/options
+        mkdir -p /tmp/options
         echo "---" > /tmp/options/options.yml
         echo "rpm_repository:" >> /tmp/options/options.yml
         echo "  digitalocean_access_token: ${{ secrets.RPM_BUILD_DO_ACCESS_TOKEN }}" >> /tmp/options/options.yml
         echo "  s3_api:" >> /tmp/options/options.yml
         echo "    access_key: ${{ secrets.RPM_BUILD_S3_ACCESS_KEY }}" >> /tmp/options/options.yml
         echo "    secret_key: ${{ secrets.RPM_BUILD_S3_SECRET_KEY }}" >> /tmp/options/options.yml
-        docker run -v /tmp/options:/root/OPTIONS:z docker.io/manageiq/rpm_build:latest build --build-type nightly --git-ref ${{ github.ref_name }} --update-rpm-repo
+    - name: Build and push RPMs
+      run: run_container_image
```

Comment:
Should we move the rest of the steps into that script too?

Reply:
I was considering that, but for now I'd like to keep it this way.  Main reason is that these secrets can't be referenced from a script, and you would need lines here anyway to do something like move them into ENV vars.  In the end, we probably don't want to be modifying the yamls like this and should use ENV vars directly in the uploader script.  For now, I just wanted to fix the issue at hand.